<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogReplace</class>
 <widget class="QDialog" name="DialogReplace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Replace text</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Text to find</string>
     </property>
     <property name="buddy">
      <cstring>editFind</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editFind"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Replace&amp;ment text</string>
     </property>
     <property name="buddy">
      <cstring>editReplace</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editReplace"/>
   </item>
   <item>
    <widget class="QCheckBox" name="checkCaseSensitive">
     <property name="text">
      <string>C&amp;ase sensitive</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkWholeWords">
     <property name="text">
      <string>&amp;Whole words only</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkFindBackward">
     <property name="text">
      <string>Find &amp;backwards</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkPrompt">
     <property name="text">
      <string>&amp;Prompt on replace</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>&amp;Replace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogReplace</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>75</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>DialogReplace</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
